src/pages/careers.js

```diff
@@ -185,7 +185,7 @@ const CareersPage = ({ location }) => {
           <h2 id='open-positions'>Open positions</h2>
           <p className='body'>
             We actively seek to broaden the diversity of our team, and strongly
-            encourage people from underrepresented groups to apply.
+            encourage people from underrepresented groups to apply. Learn more about our non-discrimination policy <a href='/non-discrimination-policy-statement.pdf'>here. (PDF, 138 KB)</a>
```

Comment:
@iris-i could we make the entire text 'non-discrimination policy' the link? Might need to reword the sentence. It would be easier then just have 'here.' be the link.

Reply:
<img width="734" alt="Screen Shot 2023-07-13 at 5 11 08 PM" src="https://github.com/CivicActions/civicactions-homesite/assets/5355315/affe790f-8536-47bf-a23f-a1f28b76a44e">
Agreed. The "(PDF size)" part makes it a little more readable but I completely agree that a more descriptive link would work better.
I can change it to 
```
<a href='/non-discrimination-policy-statement.pdf'>non-discrimination policy. (PDF, 138 KB)</a>
```
 and get rid of the "here":

Reply:
cc: @marissaleighfox